katas/langtonant/charles/langtonant.test.js

```diff
@@ -0,0 +1,167 @@
+import { createGrid } from './langtonant';
+import { initialPosition } from './langtonant';
+import { nextPosition } from './langtonant';
+import { updateValues } from './langtonant';
+import { verifyPainted } from './langtonant';
+import { changeColor } from './langtonant';
+import { antPath } from './langtonant';
+
+test('Function that creates the grid with dimension N', () => {
+    let dimension = 5;
+    let grid = Array(dimension);
+    for (let row = 0; row < dimension; row++) {
```

Comment:
Normally, it would be better to hard code the expected result instead of use code to generate it so it is easier to understand the expected result and reduce chances to add bugs in the unit tests itself. In this case, just create the matrix as
```
const expectedGrid = [
     [ {...}, {...} , {...}],
     [ {...}, {...} , {...}],
     [ {...}, {...} , {...}],
     [ {...}, {...} , {...}]
]
```